change/@fluentui-web-components-6c28acc1-a99b-4101-a17a-54d88ec1a057.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31164&commitId=bdecd69ec0b426ca66634bc28a099c44c8507a46&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button.With Long Text.default.chromium.png|1703| Changed|
|Button.With Long Text.pressed.chromium.png|1460| Changed|
|Button.With Long Text - RTL.hover.chromium.png|1607| Changed|
|Button.With Long Text - RTL.pressed.chromium.png|1445| Changed|
|Button.With Long Text.hover.chromium.png|1624| Changed|
|Button.With Long Text - RTL.default.chromium.png|1686| Changed|

</details>


  <details><summary><b>Slider</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Slider Steps.normal.chromium.png|1| Changed|
|Slider.Slider Steps Vertical.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Text</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Block.normal.chromium.png|995| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

Reply:
@chrisdholt Based on the VR tests it looks like you may have missed a rule.

Reply:
> @chrisdholt Based on the VR tests it looks like you may have missed a rule.

This needs to be rerun. The VR regression in Button got merged a little over an hour ago. With the removal of the internals, we needed to set text-align: center.